src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/InputFileResolverTests.cs

```diff
@@ -52,6 +53,38 @@ public InputFileResolverTests()
 </Project>";
         }
 
+        [Fact]
+        public void ProjectAnalyzerShouldDecodeFramework()
```

Comment:
I think these should be seperate test cases. Maybe xUnit's `[Theory]` could be used in this case.